resources/providers/awslib/securityhub/securityhub.go

```diff
@@ -38,8 +38,8 @@ type (
 )
 
 func (s SecurityHub) GetResourceArn() string {
-	if s.HubArn == nil {
-		return ""
+	if s.DescribeHubOutput == nil || s.HubArn == nil {
+		return fmt.Sprintf("securityhub - %s", s.Region)
```

Comment:
I think this would be more robust and we won't have duplication.
`return s.GetResourceName();`

Reply:
done